lib/runCall.js

```diff
@@ -259,26 +259,33 @@ module.exports = function (opts, cb) {
           stateManager.putAccount(address, stateAccount, _callback)
         })
       },
-      function (_callback) {
-        if (code.length !== 0) {
-          stateManager.putContractCode(address, code, _callback)
-        } else {
-          _callback()
-        }
-      },
-      function (_callback) {
-        async.series(
-          storageUpdateList.map(item => putContractHelper(address, item)),
-          _callback
-        )
-      }], _done)
+        function (_callback) {
+          if (code.length !== 0) {
+            stateManager.putContractCode(address, code, _callback)
+          } else {
+            _callback()
+          }
+        },
+        function (_callback) {
+          async.series(
+            storageUpdateList.map(item => putContractHelper(address, item)),
+            _callback
+          )
+        }], _done)
   }
 
   function putContractHelper (address, item) {
     return function (_callback) {
       var key = Buffer.from(item.getOffset())
       var value = Buffer.from(item.getData())
       console.log('Setting contract (' + address.toString('hex') + '): ' + JSON.stringify(key) + ' (' + parseInt(value.toString('hex'), 16) + ')')
+      // Remove any zeroes prepended by KEVM
+      while (key[0] === 0 && key.length > 1) {
```

Comment:
If we know these are addresses, maybe we should just be taking the lower 20 bytes instead of chopping off any leading zeros. Not sure whether ethereum addresses starting with 0 are considered valid, but there isn't anything stopping KEVM from generating an address which starts with 0.